hcl/eval/partial_fuzz_test.go

```diff
@@ -92,13 +90,9 @@ func FuzzPartialEval(f *testing.F) {
 
 		const testattr = "attr"
 
-		cfg := hcldoc(
```

Comment:
Now hclwriter always produces valid HCL, so if you use it to build invalid HCL files it panics, seemed like the way to go since it is what we want on 99% of the tests, we use it to build some valid HCL file for testing. But here we generate possibly invalid HCLs and that would cause panics, so it seemed easier to just create the HCL with a string since it is a pretty straithforward attr = val.